packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -515,6 +516,54 @@ class HorizontalDragGestureRecognizer extends DragGestureRecognizer {
   String get debugDescription => 'horizontal drag';
 }
 
+// A horizontal drag gesture detector that only responds to events on the
+// scrollbar's thumb and ignores everything else.
+// TODO(justinmc): I wanted to put this in cuperinto/scrollbar.dart along with
+// the long press detector, but it seems I can't override _isFlingGesture
```

Comment:
Why not make _isFlingGesture public? AFAICT the implementations are all trivial.